elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -991,6 +991,16 @@ elytron.failover-realm.delegate-realm=Reference to the security realm to use as
 elytron.failover-realm.failover-realm=Reference to the security realm to use in case the delegate-realm is unavailable.
 elytron.failover-realm.emit-events=Whether SecurityEvent signifying delegate-realm unavailability should be emitted.
 
+elytron.jaas-realm=A JAAS realm definition. It wraps a LoginContext initialized from the provided jaas-config-path.
+# Operations
+elytron.jaas-realm.add=The add operation for the security realm.
+elytron.jaas-realm.remove=The remove operation for the security realm.
+elytron.jaas-realm.callback-handler=Callback handler to use with the LoginContext.
```

Comment:
It could be useful to document here what can be used when this is not defined.

Reply:
Added